### ISSUE TITLE
Issue using flag-icon-css

### ISSUE THREAD

**Issue by mfirry** [webpack-contrib/file-loader#155]
I've got a pretty basic react app that uses webpack 1.13.3 and file-loader 0.9.0 and [flag-icon-css](https://github.com/lipis/flag-icon-css).

In my `webpack.config.js` I've got an alias like this:

`flag_icon_css: __dirname + "/node_modules/flag-icon-css/css/flag-icon.css"`

which allows me to have this in my `Main.js`

`import flagIconCss from 'flag_icon_css'`

*flag-icon-css* has this [flags](https://github.com/lipis/flag-icon-css/tree/master/flags) folder full with `svg` images.

The rule is use is:
```
      {
        test: /\.(ttf|eot|svg|woff|woff2)(\?v=[0-9]\.[0-9]\.[0-9])?$/,
        loader: 'file-loader'
      }
```

I don't have anything else specific for svg files.

What happens now is that in my build I end up having hundreds of `bfaad5727ebb4257ba8f21ab3e7a1b5a`-like `svg` files while all I'd really want is to have the very same `flags` folder copied into my dist folder.

I'm fairly new to webpack, so any hint into the right direction would be a great help.

**Comment by bebraw** [comment]
Maybe [webpack-svg-sprite-plugin](https://www.npmjs.com/package/webpack-svg-sprite-plugin) would work better for your purposes. It's not a great idea to end up with lots of small SVG files to load. You can also try **url-loader** and then inline the SVGs into the resulting CSS. That will bloat it a bit but you'll avoid a request per each SVG.

**Comment by devotis** [comment]
@mfirry The [svg-url-loader](https://github.com/bhovhannes/svg-url-loader) solved it for me. Just `npm install --save-dev svg-url-loader` and add this rule to `webpack.config.js` > module.rules

```javascript
{
  test: /\.svg$/,
  loader: 'svg-url-loader'
}
```

**Comment by bebraw** [comment]
Ok, time to close. Thanks.

**Comment by scottyh527** [comment]
Hi I'm having a similar issue I think. I recently tried implementing webpack, and the icons that I had before of the flags don't seem to show up. I have a app.scss file where i'm importing these icons 

// Styles
// CoreUI Icons Set
@import '~@coreui/icons/css/coreui-icons.css';
// Import Flag Icons Set
@import '~flag-icon-css/css/flag-icon.min.css';
// Import Font Awesome Icons Set
@import '~font-awesome/css/font-awesome.min.css';
// Import Simple Line Icons Set
@import '~simple-line-icons/css/simple-line-icons.css';
// Import Main styles for this application
@import './scss/style.scss';

And in app.js importing the scss file. The other icons seem to be displaying correctly. 

This is what I have for my webpack config. 

{
        test: /\.svg$/,
        loader: 'svg-url-loader'
      },
      {
        test: /\.(png|svg|jpg|gif|jpeg|webp)$/, use: {
          loader: 'file-loader',
          options: { name: '[name].[ext]' }
        }
      },
      { test: /\.woff($|\?)|\.woff2($|\?)|\.ttf($|\?)|\.eot($|\?)|\.svg($|\?)/, use: ['file-loader'] }

I tried the svg loader and the webpack svg sprite plugin but the flags still won't display, any help would be appreciated.
